config.js

```diff
@@ -193,6 +193,19 @@ config.search = {
                         'type': 'custom',
                         'tokenizer': 'letter',
                         'filter': ['lowercase', 'content_edgengram']
+                    },
+                    'display_name': {
```

Comment:
@simong: Would you be able to add a test for this functionality (e.g. verify that `Simon Gaeremynck` is shown before `Simon The Great` when searching for `Simon G`) ?